screens/loginScreen.tsx

```diff
@@ -0,0 +1,48 @@
+import { createUserWithEmailAndPassword, signInWithEmailAndPassword  } from 'firebase/auth';
+import React from 'react';
+import { useState } from 'react';
+import { View, Text, Button } from 'react-native';
+import { TextInput } from 'react-native-gesture-handler';
+import { DefaultScreenProps } from '../common/DefaultScreenProps';
+import { auth } from '../firebase/firebase.config';
+import { globalStyles } from '../styles/global';
+
+export default function LoginScreen({ navigation }: DefaultScreenProps) {
+
+  const [email, setEmail] = useState("")
+  const [password, setPassword] = useState("")
+
+  function signIn(){
+    signInWithEmailAndPassword(auth, email, password)
+    .then((res) => {
+      console.log(res.user.getIdToken)
```

Comment:
Zostawiamy na razie? xD

Reply:
No to tak na razie xD Trzeba i tak coś zrobić żeby to zapisać a to taki placeholder w sumie